semanticdb/metacp/src/main/scala/scala/meta/internal/metacp/Javacp.scala

```diff
@@ -0,0 +1,402 @@
+package scala.meta.internal.metacp
+
+import java.nio.file.Files
+import java.nio.file.Path
+import scala.collection.JavaConverters._
+import scala.collection.mutable.ArrayBuffer
+import scala.collection.mutable.ListBuffer
+import scala.meta.internal.metacp.asm._
+import scala.meta.internal.metacp.asm.JavaTypeSignature._
+import scala.meta.internal.semanticdb3.SymbolInformation.{Kind => k}
+import scala.meta.internal.{semanticdb3 => s}
+import scala.tools.asm.ClassReader
+import scala.tools.asm.tree.ClassNode
+import scala.tools.asm.tree.FieldNode
+import scala.tools.asm.tree.MethodNode
+import scala.tools.asm.{Opcodes => o}
+
+object Javacp {
+
+  def sdocument(root: Path, file: Path, scopes: TypeVariableScopes): s.TextDocument = {
```

Comment:
Typically, a collection of scopes is referred to using a dedicated name. In Rsc, we're saying `Env`. In Scalac and Dotty, it's `Context`.

Reply:
Done.